src/detect-pcre.h

```diff
@@ -42,6 +42,8 @@
 typedef struct DetectPcreData_ {
     DetectParseRegex parse_regex;
     int thread_ctx_id;
+    /* String for logging */
+    char *pcre_str;
```

Comment:
I want to avoid things like this. The match structures need to be as minimal and "lean" as possible for the matching. This type of data isn't used at matching and shouldn't affect the memory layout.